### PR TITLE
refactor: properties components

### PR DESCRIPTION
**Pull request by GeronimoJohn**
# Description

Refactor journey view properties. Moved out access control and journey URL to their own components

[- Link to Basecamp Todo](https://3.basecamp.com/3105655/buckets/28980303/todos/5251462882)

# How should this PR be QA Tested?

- [ ] Access Control to work as expected
- [ ] Journey URL to work as expected

# Checklist

- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] Any dependent changes have been merged into main

### CODE REVIEW

File: apps/journeys-admin/src/components/JourneyView/Properties/JourneyURL/JourneyURL.spec.tsx [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+import { SnackbarProvider } from 'notistack'
+import { defaultJourney } from '../../data'
+import { JourneyURL } from './JourneyURL'
+
+jest.mock('@mui/material/useMediaQuery', () => ({
+  __esModule: true,
+  default: () => true
+}))
+
+describe('Journey URL', () => {
```
Comment (storyworks) [inline apps/journeys-admin/src/components/JourneyView/Properties/JourneyURL/JourneyURL.spec.tsx:13]: should be name of component

File: apps/journeys-admin/src/components/JourneyView/JourneyView.tsx [hunk 2]
```diff
@@ -80,20 +80,19 @@ export function JourneyView(): ReactElement {
           <CopyTextField
             value={
               journey?.slug != null
-                ? `${
-                    process.env.NEXT_PUBLIC_JOURNEYS_URL ??
-                    'https://your.nextstep.is'
-                  }/${journey.slug}`
+                ? `${process.env.NEXT_PUBLIC_JOURNEYS_URL ??
```
Comment (storyworks) [inline apps/journeys-admin/src/components/JourneyView/JourneyView.tsx:83]: Did prettier not run? How is formatting different?

Comment (GeronimoJohn) [reply apps/journeys-admin/src/components/JourneyView/JourneyView.tsx:83]: Sometimes my prettier acts up and messes up formating

File: apps/journeys-admin/src/components/JourneyView/Properties/AccessControl/AccessControl.spec.tsx [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+    const { getByText } = render(
+      <SnackbarProvider>
+        <MockedProvider>
+          <JourneyProvider value={{ journey: defaultJourney, admin: true }}>
+            <AccessControl />
+          </JourneyProvider>
+        </MockedProvider>
+      </SnackbarProvider>
+    )
+    expect(getByText('Access Control')).toBeInTheDocument()
```
Comment (storyworks) [inline apps/journeys-admin/src/components/JourneyView/Properties/AccessControl/AccessControl.spec.tsx:24]: Could add id to AccessAvatar and check that it displays as well?

File: apps/journeys-admin/src/components/JourneyView/Properties/Properties.spec.tsx [hunk 4]
```diff
@@ -10,9 +10,9 @@ jest.mock('@mui/material/useMediaQuery', () => ({
   default: () => true
 }))
 
-describe('JourneyView/Properties', () => {
-  it('should handle edit journey slug', async () => {
-    const { getAllByRole, getByRole, queryByRole } = render(
+describe('JourneyView', () => {
```
Comment (storyworks) [inline apps/journeys-admin/src/components/JourneyView/Properties/Properties.spec.tsx:13]: keep as Properties or JourneyView/Properties if we have another Properties component..

**Pull request merged**